forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main/java/com/forgerock/openbanking/aspsp/rs/store/api/openbanking/vrp/v3_1_8/DomesticVrpConsentsApi.java

```diff
@@ -0,0 +1,150 @@
+    @RequestMapping(
+            value = DOMESTIC_VRP_CONSENTS,
+            produces = {"application/json"},
+            method = POST)
+    ResponseEntity<OBDomesticVRPConsentResponse> createDomesticVrpConsent(
+            @ApiParam(value = "Default", required = true)
+            @Valid
+            @RequestBody OBDomesticVRPConsentRequest domesticVRPConsentRequest // added manually
+            // none of the headers have been generated from the open-api spec
+    ) throws OBErrorResponseException;
+
+    /**
+     * GET /domestic-vrp-consents/:consentId : create a domestic vrp
+     * create a domestic vrp
+     *
+     * @return (status code 200)
+     */
+    @ApiOperation(
+            value = "create a domestic vrp",
+            nickname = "domesticVrpConsentsConsentIdGet",
+            notes = "create a domestic vrp",
+            response = OBDomesticVRPConsentResponse.class,
+//            authorizations = {
+//                    @Authorization(value = "TPPOAuth2Security", scopes = {
+//                            @AuthorizationScope(scope = "vrps", description = "Generic VRPs scope")
+//                    })
+//            },
+            tags = {"variable-recurring-payments",})
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "", response = OBDomesticVRPConsentResponse.class)})
+    @RequestMapping(
+            value = DOMESTIC_VRP_CONSENTS + "/:consentId",
+            produces = {"application/json"},
+            method = RequestMethod.GET)
+    ResponseEntity<OBDomesticVRPConsentResponse> getDomesticVrpConsent() throws OBErrorResponseException;
+
+    /**
+     * DELETE /domestic-vrp-consents/:consentId : create a domestic vrp
+     * create a domestic vrp
+     *
+     * @return all good (status code 204)
+     */
+    @ApiOperation(
+            value = "create a domestic vrp",
+            nickname = "domesticVrpConsentsConsentIdDelete",
+            notes = "create a domestic vrp",
+//            authorizations = {
+//                    @Authorization(value = "TPPOAuth2Security", scopes = {
+//                            @AuthorizationScope(scope = "vrps", description = "Generic VRPs scope")
+//                    })
+//            },
+            tags = {"variable-recurring-payments",})
+    @ApiResponses(value = {@ApiResponse(code = 204, message = "all good")})
+    @RequestMapping(value = DOMESTIC_VRP_CONSENTS + "/:consentId", method = DELETE)
+    ResponseEntity<Void> deleteDomesticVrpConsent() throws OBErrorResponseException;
+
+    /**
+     * POST /domestic-vrp-consents/:consentId/funds-confirmation : create a domestic vrp
+     * create a domestic vrp
+     *
+     * @return (status code 200)
+     */
+    @ApiOperation(
+            value = "create a domestic vrp",
+            nickname = "domesticVrpConsentsConsentIdFundsConfirmationPost",
+            notes = "create a domestic vrp",
+            response = Object.class,
+//            authorizations = {
```

Comment:
Why are all the authorization's commented out?

Reply:
Because I don't know if they're correct. I mentioned _"I have inserted them in commented out sections, but certain things (e.g. the scopes) may not be correct"_ in the description above.